### PR TITLE
Code Quality: Moved service interfaces to Contracts & Removed JumpListHelper

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

As the title.

## PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Related #14572
- [ ] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   _None_

## Screenshots

_None_

### CODE REVIEW

**Review by yaira2 — DISMISSED** [review]

File: src/Files.App/Services/WindowsJumpListService.cs [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
 	{
 		private const string JumpListRecentGroupHeader = "ms-resource:///Resources/JumpListRecentGroupHeader";
 		private const string JumpListPinnedGroupHeader = "ms-resource:///Resources/JumpListPinnedGroupHeader";
 
+		public async Task InitializeAsync()
+		{
+			try
+			{
+				App.QuickAccessManager.UpdateQuickAccessWidget -= UpdateQuickAccessWidgetAsync;
+				App.QuickAccessManager.UpdateQuickAccessWidget += UpdateQuickAccessWidgetAsync;
```
Comment (yaira2) [inline src/Files.App/Services/WindowsJumpListService.cs:21]: I don't think this belongs in the jump list service, this is related to quick access.

Comment (0x5bfa) [reply src/Files.App/Services/WindowsJumpListService.cs:21]: This provides user to refresh jump list when user edits quick access widget pinned items.
I'll make it more appropriate event method name.

Comment (yaira2) [reply src/Files.App/Services/WindowsJumpListService.cs]: I see, thanks for the clarification.

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**